src/test.rs

```diff
@@ -171,9 +172,9 @@ fn simple_blurred_quad() {
         for y in 0..5 {
             scene.add_quad(
                 Quad::new(
-                    vec2(15., 15.) + vec2(x as f32 * 60., y as f32 * 60.),
-                    vec2(50., 50.),
-                    vec4(x as f32 / 5., y as f32 / 5., 1., 1.),
+                    point2!(15., 15.) + vec2!(x as f32 * 60., y as f32 * 60.),
+                    size2!(50., 50.),
+                    Srgba::new(x as f32 / 5., y as f32 / 5., 1., 1.),
```

Comment:
I think you could do this interpolation in linear space, and then convert to sRGB to avoid the regression.

https://blog.johnnovak.net/2016/09/21/what-every-coder-should-know-about-gamma/#gradients

Reply:
I'm actually not worried about it. I'm going to leave it as is as the image demonstrates that srgb is actually being used.